sdk/src/main/kotlin/au/kinde/sdk/KindeSDK.kt

```diff
@@ -220,10 +220,6 @@ class KindeSDK(
             .build()
         val endSessionIntent = authService.getEndSessionRequestIntent(endSessionRequest)
         endTokenLauncher.launch(endSessionIntent)
-
-        apiClient.setBearerToken("")
-        sdkListener.onLogout()
-        store.clearState()
```

Comment:
@mnghn07 With the removal or the state and Token being cleared, are these sufficiently cleared on logout still?

Reply:
Hi @DanielRivers , yes it's still sufficiently cleared on logout. Since these functions were already called on `endTokenLauncher`.